README.md

```diff
@@ -428,6 +428,21 @@ Finally, restart Postfix
 sudo systemctl restart postfix
 ```
 
+#### [Optional] Enable TLS on Postfix submission
+
+In case your Postfix server is on another server, it's recommended to enable TLS on Postfix submission to 
+secure the connection between SimpleLogin email handler and Postfix.
+
+This can be enabled by adding those lines at the end of `/etc/postfix/master.cf`
```

Comment:
And setting the `POSTFIX_SUBMISSION_TLS` to `true` in the .env file.

Reply:
Good call! I just updated the PR to include that and move Postfix-TLS instructions to its own file.